src/legacy/core_plugins/kibana/public/discover/embeddable/_index.scss

```diff
@@ -0,0 +1,11 @@
+  /**
+   * 1. We want the kbnDocTable__container to scroll only when embedded in an embeddable panel
+   * 2. Force a better looking scrollbar
+   */
+.embPanel {
+   .kbnDocTable__container {
```

Comment:
This style block isn't being applied. You'll need to move these styles into a `_embeddable.scss` file which would get imported here, then import this index file in https://github.com/elastic/kibana/blob/9cf5e2c9f65dc5f60a76cf8c03cea5e6377e687f/src/legacy/core_plugins/kibana/public/discover/_index.scss

Reply:
Good catch!